README.md

```diff
@@ -156,7 +183,7 @@ Fzf optionally comes with its own [fish extension][]. It is substantial but `fzf
 [fisher]: https://github.com/jorgebucaran/fisher
 [fzf_default_opts]: https://github.com/junegunn/fzf#environment-variables
 [fzf]: https://github.com/junegunn/fzf
-[git log]: images/git_log.gif
+[git log search]: images/git_log.gif
```

Comment:
Realized all the other gif link names had an action verb at the end so make the git log link name follow suit.